frontend/src/setupTests.ts

```diff
@@ -3,3 +3,7 @@
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
 import "@testing-library/jest-dom";
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+//@ts-ignore
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+window.URL.createObjectURL = function () {};
```

Comment:
Why is this needed?

Reply:
so basically when we run the test we have to call this  `createObjectURL`. otherwise test will fail.

Reply:
<img width="527" alt="Screen Shot 2022-08-01 at 3 44 28 PM" src="https://user-images.githubusercontent.com/13723264/182232307-3115442e-6ecc-4990-a1a2-5cc1205ac67f.png">
gotcha, I was able to remove the ts-ignore by doing this

Reply:
also the above function doesn't need the `no-empty-function` eslint ignore so that can also be removed